shell/app/common/components/table/index.tsx

```diff
@@ -62,15 +62,17 @@ function WrappedTable<T extends object = any>({
 }: IProps<T>) {
   const [columns, setColumns] = React.useState<Array<ColumnProps<T>>>(allColumns);
   const [sort, setSort] = React.useState<SorterResult<T>>({});
-  const sortCompareRef = React.useRef() as { current: (a: T, b: T) => number };
+  const sortCompareRef = React.useRef() as { current: ((a: T, b: T) => number) | null };
   const [defaultPagination, setDefaultPagination] = React.useState<TablePaginationConfig>({
     current: 1,
     total: dataSource.length,
     ...PAGINATION,
   });
+  const isWebPaging = !(paginationProps && paginationProps.current) && paginationProps !== false; // Determine whether front-end paging
```

Comment:
`isWebPaging ` confused me that I thought is there any mobile paging. Rename to isFrontendPaging is better